include/oxen/quic/address.hpp

```diff
@@ -340,6 +342,12 @@ namespace oxen::quic
             return *this;
         }
 
+        auto operator<=>(const Path& other) const { return std::tie(local, remote) <=> std::tie(other.local, other.remote); }
+
+        bool operator==(const Path& other) const { return (*this <=> other) == 0; }
+
+        bool operator!=(const Path& other) const { return !(*this == other); }
```

Comment:
With C++20, you can drop the `operator !=` and get it automatically via `operator ==`.

Reply:
Great! Will drop the one on `Address` too, was just going for uniformity :+1: